.github/workflows/deploy-testing-subgraph.yaml

```diff
@@ -2,21 +2,21 @@ name: Deploy subgraph to testing env
 
 on:
   workflow_dispatch:
-    branches: [main]
```

Comment:
why dont we keep this to force redeploying?

Reply:
IMHO the `branches` option on the `workflow_dispatch` event is not correct (never mentionned in the GH docs: https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#workflow_dispatch).
The idea having `workflow_dispatch` is to allow running the job with a manual trigger and it's always possible to choose the branch there.

Reply:
Note: the other event is the one that is trigerring the job when there is a push on main branch with changes in the subgraph package
```
push:
    branches:
      - 'main'
    paths:
      - packages/subgraph/**
```